README.md

```diff
@@ -6,6 +6,9 @@ Documentation|[![Documentation Status](https://readthedocs.org/projects/dexterou
 Build|[<img src="https://codebuild.eu-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoibGw4V2ovMmRXUGdCYlJwd0xzbVlGR3JtQVE3TytlTm9uenJ2bURyZ3lQc1prRVRqa1BJUk1Qb0d4TmhaYTRBNGJyazBlSU5zT0NyMEhNOTJMRlpYUVZ3PSIsIml2UGFyYW1ldGVyU3BlYyI6IkNMcWZBM01LRDlRUXl0eDgiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=noetic-devel"/>](https://eu-west-2.console.aws.amazon.com/codesuite/codebuild/projects/auto_sr_interface_noetic-devel_install_check/)
 Style|[<img src="https://codebuild.eu-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiRmh2c2hJaFZUZWh3REJNUDkzYUxXbm45bUs2Umo3UzAvQ096dThJRlR3SVdmN1ZXZE9Kczc4M005SGhOeGJKUVFXVnRIV1o3bjM3aTJicWlsbEZ0WDJ3PSIsIml2UGFyYW1ldGVyU3BlYyI6Im0zUmUyd0RKbldiekVyUzgiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=noetic-devel"/>](https://eu-west-2.console.aws.amazon.com/codesuite/codebuild/projects/auto_sr_interface_noetic-devel_style_check/)
 Code Coverage|[<img src="https://codebuild.eu-west-2.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiUUdRZlVJMFAzQ2FVMEp0K2tDOVlmUGoxNWY0aVFTZHpmVFpDWXFadkJaUGRMRndmZllxeDJMaTBxbmhteFh1SlZzWFcrTWlnRFFRdjNmdFVNckhLczFBPSIsIml2UGFyYW1ldGVyU3BlYyI6ImNrbHdUT21USS9zS3l5ME8iLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=noetic-devel"/>](https://eu-west-2.console.aws.amazon.com/codesuite/codebuild/projects/auto_sr_interface_noetic-devel_code_coverage/)
+Codecov Report |[![codecov](https://codecov.io/gh/shadow-robot/sr_interface/branch/kinetic-devel/graph/badge.svg?token=BFvwh3wAgu)](https://codecov.io/gh/shadow-robot/sr_interface)
```

Comment:
should be probably noetic-devel and added also to codecov? (great thing to have this!)